src/megaclient.cpp

```diff
@@ -4662,8 +4665,23 @@ bool MegaClient::procsc()
 
                     if (!insca_notlast)
                     {
+                        sc_checkSequenceTag(string());
                         app->catchup_result();
                     }
+
+                    if (pendingsccommit && sctable && !reqs.cmdsInflight())
+                    {
+                        sctable->commit();
+                        sctable->begin();
+                        app->notify_dbcommit();
+                        pendingsccommit = false;
+                    }
+
+                    if (pendingsccommit)
+                    {
+                        LOG_debug << "Postponing DB commit until cs requests finish";
```

Comment:
We can change the message - it will have the source file and line number though.  I added this one because it's useful to see if we process several sc batches without the cs completing.

Reply:
Yes, no objection to the log message. It's just that depending on the logger, IIRC, the source file might not be added? not sure. We always try to make different log messages here and there :)

Reply:
I'll add a minor change to the message.